electricpy/visu.py

```diff
@@ -321,4 +325,209 @@ def get_output_power(self):
         # low slip
         p_x_2 = center_x - self.radius * _np.cos(beta_1)
         p_y_2 = center_y + self.radius * _np.sin(beta_1)
-        return [p_x_1, p_y_1], [p_x_2, p_y_2]
\ No newline at end of file
+        return [p_x_1, p_y_1], [p_x_2, p_y_2]
+
+class PowerCircle:
+    r"""
+    Plot Power Circle Diagram of Transmission System.
+
+    This class is designed to plot the power circle diagram of a transmission
+    system both sending and reciving ends.
+
+    Parameters
+    ----------
+    type: str "sending" or "receiving"
+
+    Vr: complex Receiving End Voltage
+
+    Vs: complex Sending End Voltage
+
+    power_factor: float Power Factor
+
+    Pr: float Receiving End Real Power
+
+    Qr: float Receiving End Reactive Power
+
+    Sr: float Receiving End Total Power
+
+    Ps: float Sending End Real Power
+
+    Qs: float Sending End Reactive Power
+
+    Ss: float Sending End Total Power
+
+    power_factor: float Power Factor
+
+    A: float ABCD Matrix
```

Comment:
Could we elaborate on `ABCD Matrix`? At first glance, I don't know what it's intended to provide, and that's one of the benefits of the builtin-documentation, is that the documentation can act, in part, as an educational resource to help others learn more about the formulas as well as how they're used.